OpenBlas is failing to build within MXE

And just as I thought all issues were behind me, this one popped up.  For some reason, there is no log file.  Here is the error message from make:

[build]     openblas               i686-w64-mingw32.static.posix

Failed to build package openblas for target i686-w64-mingw32.static.posix!
------------------------------------------------------------
make[2]: Entering directory '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
Makefile.system:412: *** OpenBLAS: Cannot set both USE_OPENMP=1 and USE_THREAD=0. The USE_THREAD=0 is only for building single thread version..  Stop.
make[2]: Leaving directory '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
Makefile:687: recipe for target 'build-only-openblas_i686-w64-mingw32.static.posix' failed
make[1]: *** [build-only-openblas_i686-w64-mingw32.static.posix] Error 2
make[1]: Leaving directory '/home/brian/mxe-git'
real	0m5.993s
user	0m4.524s
sys	0m0.588s
------------------------------------------------------------
[log]      /home/brian/mxe-git/log/openblas_i686-w64-mingw32.static.posix

Makefile:687: recipe for target '/home/brian/mxe-git/usr/i686-w64-mingw32.static.posix/installed/openblas' failed
make: *** [/home/brian/mxe-git/usr/i686-w64-mingw32.static.posix/installed/openblas] Error 1

Found the actual log:
```make[1]: Entering directory '/home/brian/mxe-git'
uname -a
Linux brian-VirtualBox 4.10.0-37-generic #41~16.04.1-Ubuntu SMP Fri Oct 6 22:42:59 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
git log --pretty=tformat:"%H - %s [%ar] [%d]" -1
9a9b9f6515317d483e9179eaee85fbd62c64f4e0 - hyperscan: fix build where native SSSE3 detection fails [3 days ago] [ (HEAD -> master)]
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
autoconf --version 2>/dev/null | head -1
autoconf (GNU Autoconf) 2.69
automake --version 2>/dev/null | head -1
automake (GNU automake) 1.15
gcc --version
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.5) 5.4.0 20160609
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

g++ --version
g++ (Ubuntu 5.4.0-6ubuntu1~16.04.5) 5.4.0 20160609
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

python --version
Python 2.7.12
perl --version 2>&1 | head -3

This is perl 5, version 22, subversion 1 (v5.22.1) built for x86_64-linux-gnu-thread-multi
(with 58 registered patches, see perl -V for more detail)
rm -rf   '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix'
mkdir -p '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix'
mkdir -p '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
mkdir -p '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20.build_'
# disable wine with readonly directory
# see https://github.com/mxe/mxe/issues/841
mkdir -p '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/readonly'
chmod 0555 '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/readonly'
cd '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix' &&  tar xzf '/home/brian/mxe-git/pkg/openblas-v0.2.20.tar.gz'
cd '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
(cd '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20' && patch -p1 -u) < /home/brian/mxe-git/src/openblas-1-fixes.patch
patching file Makefile
make -C '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20' -j '1' PREFIX='/home/brian/mxe-git/usr/i686-w64-mingw32.static.posix' CROSS_SUFFIX='i686-w64-mingw32.static.posix-' FC='i686-w64-mingw32.static.posix-gfortran' CC='i686-w64-mingw32.static.posix-gcc' HOSTCC='gcc' MAKE_NB_JOBS=-1 CROSS=1 BUILD_RELAPACK=1 USE_THREAD=1 USE_OPENMP=1 TARGET=CORE2 DYNAMIC_ARCH=1 ARCH=x86 BINARY=32 NO_SHARED=1  EXTRALIB="`'i686-w64-mingw32.static.posix-pkg-config' --libs pthreads` -fopenmp -lgfortran -lquadmath"
make[2]: Entering directory '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
Makefile.system:412: *** OpenBLAS: Cannot set both USE_OPENMP=1 and USE_THREAD=0. The USE_THREAD=0 is only for building single thread version..  Stop.
make[2]: Leaving directory '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
Makefile:687: recipe for target 'build-only-openblas_i686-w64-mingw32.static.posix' failed
make[1]: *** [build-only-openblas_i686-w64-mingw32.static.posix] Error 2
make[1]: Leaving directory '/home/brian/mxe-git'

real	0m9.874s
user	0m4.592s
sys	0m1.040s```

and a new copy of the original error message:

```[build]     openblas               i686-w64-mingw32.static.posix

Failed to build package openblas for target i686-w64-mingw32.static.posix!
------------------------------------------------------------
make[2]: Entering directory '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
Makefile.system:412: *** OpenBLAS: Cannot set both USE_OPENMP=1 and USE_THREAD=0. The USE_THREAD=0 is only for building single thread version..  Stop.
make[2]: Leaving directory '/home/brian/mxe-git/tmp-openblas-i686-w64-mingw32.static.posix/OpenBLAS-0.2.20'
Makefile:687: recipe for target 'build-only-openblas_i686-w64-mingw32.static.posix' failed
make[1]: *** [build-only-openblas_i686-w64-mingw32.static.posix] Error 2
make[1]: Leaving directory '/home/brian/mxe-git'
real	0m5.993s
user	0m4.524s
sys	0m0.588s
------------------------------------------------------------
[log]      /home/brian/mxe-git/log/openblas_i686-w64-mingw32.static.posix

Makefile:687: recipe for target '/home/brian/mxe-git/usr/i686-w64-mingw32.static.posix/installed/openblas' failed
make: *** [/home/brian/mxe-git/usr/i686-w64-mingw32.static.posix/installed/openblas] Error 1```

--- a/src/openblas.mk
+++ b/src/openblas.mk
@@ -23,6 +23,7 @@ $(PKG)_MAKE_OPTS = \
         BUILD_RELAPACK=1 \
         USE_THREAD=1 \
         USE_OPENMP=1 \
+        NUM_THREADS=$(call LIST_NMAX, 2 $(NPROCS)) \
         TARGET=CORE2 \
         DYNAMIC_ARCH=1 \
         ARCH=$(strip \

